src/main/java/com/plaid/client/response/AssetReportGetResponse.java

```diff
@@ -0,0 +1,181 @@
+  }
+
+  public final static class Account {
+    private String accountId;
+    private String mask;
+    private String name;
+    private String officialName;
+    private String type;
+    private String subtype;
+    private List<Owner> owners;
+    private Balance balances;
+    private List<HistoricalBalance> historicalBalances;
+    private List<Transaction> transactions;
+    private Integer daysAvailable;
+
+    public String getAccountId() { return accountId; }
+    public String getMask() { return mask; }
+    public String getName() { return name; }
+    public String getOfficialName() { return officialName; }
+    public String getType() { return type; }
+    public String getSubtype() { return subtype; }
+    public List<Owner> getOwners() { return owners; }
+    public Balance getBalances() { return balances; }
+    public List<HistoricalBalance> getHistoricalBalances() { return historicalBalances; }
+    public List<Transaction> getTransactions() { return transactions; }
+    public Integer getDaysAvailable() { return daysAvailable; }
+  }
+
+  public final static class Owner {
+    private List<String> names;
+    private List<PhoneNumber> phoneNumbers;
+    private List<Email> emails;
+    private List<Address> addresses;
+
+    public List<String> getNames() { return names; }
+    public List<PhoneNumber> getPhoneNumbers() { return phoneNumbers; }
+    public List<Email> getEmails() { return emails; }
+    public List<Address> getAddresses() { return addresses; }
+  }
+
+  public final static class PhoneNumber {
+    private String data;
+    private boolean primary;
+    private String type;
+
+    public String getData() { return data; }
+    public boolean isPrimary() { return primary; }
+    public String getType() { return type; }
+  }
+
+  public final static class Email {
+    private String data;
+    private boolean primary;
+    private String type;
+
+    public String getData() { return data; }
+    public boolean isPrimary() { return primary; }
+    public String getType() { return type; }
+  }
+
+  public final static class Address {
+    private AddressData data;
+    private boolean primary;
+
+    public AddressData getData() { return data; }
+    public boolean isPrimary() { return primary; }
+  }
+
+  public final static class AddressData {
+    private String city;
+    private String state;
+    private String street;
+    private String zip;
+
+    public String getCity() { return city; }
+    public String getState() { return state; }
+    public String getStreet() { return street; }
+    public String getZip() { return zip; }
+  }
+
+  public final static class Balance {
+    private Double avaliable;
+    private Double current;
+
+    public Double getAvaliable() { return avaliable; }
+    public Double getCurrent() { return current; }
+  }
+
+  public final static class HistoricalBalance {
+    private Date date;
+    private Double current;
```

Comment:
Sorry I didn't mention this earlier, but there are two additional fields on `HistoricalBalance`, `Transaction`, and `Balance`: `unofficial_currency_code` and `iso_currency_code`. See our docs [here](https://plaid.com/docs/api/#transactions) for a definition of each. They are both strings.